ccd-definition/AuthorisationCaseField/gatekeeper.json

```diff
@@ -5,5 +5,12 @@
     "CaseFieldID": "allocationDecision",
     "UserRole": "caseworker-publiclaw-gatekeeper",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "allocationProposal",
+    "UserRole": "caseworker-publiclaw-gatekeeper",
+    "CRUD": "CRU"
```

Comment:
Should the gatekeeper add the allocation proposal?

Reply:
So i added this CRU back in as it looks like some things got removed from master, ^ the permissions, which meant that when i went onto allocation decision page i was seeing this bug >> "The proposed allocation is ${allocationProposal.proposalReason}" (Where it wasn't pulling through the proposal reason".. I added this again as i knew this was what fixed it

Reply:
changed to R